src/pacmod3_node.cpp

```diff
@@ -88,77 +88,77 @@ LNI::CallbackReturn PACMod3Node::on_configure(const lc::State & state)
   pub_can_rx_ = this->create_publisher<can_msgs::msg::Frame>("can_rx", 100);
 
   can_pubs_[GlobalRptMsg::CAN_ID] =
-    this->create_publisher<pacmod_msgs::msg::GlobalRpt>("parsed_tx/global_rpt", 20);
```

Comment:
add `using namespace pacmod3_msgs::msg;` to save typing of all these `pacmod3_msgs::msg`

Reply:
I thought that was generally discouraged. There's a note about it here:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#to-do-unclassified-proto-rules
but it's not an official rule.
In this case it's probably pretty safe to do since It's just ROS messages, however I think it's useful to distinguish between the msg classes defined in `pacmod3_msgs` and the custom CAN msg classes defined in this driver for parsing reasons (for example `GlobalRptMsg`);

Reply:
It is indeed generally discouraged for the reason of potential name clash. But in this small repo, almost every msg definition from that namespace is used and they are pretty self-isolated. It should be fine to import the whole namespace. But it is fine to keep it as is to differentiate from the CAN msg definitions.